docs/themes/sphinx_rtd_theme/theme.conf

```diff
@@ -18,3 +18,4 @@ prev_next_buttons_location = bottom
 style_external_links = False
 style_nav_header_background =
 vcs_pageview_mode =
+mamedevorg = True
```

Comment:
This file is missing the final newline.

Reply:
Noticed it right after the PR was made. I'll go ahead and do corrections next chance I get to sit down and work on it.